### DISCUSSION TITLE
Docker Hub Tags not up to date

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#9808] [General]
I found multiple tags in the docker hub of netbox that are not up to date:

```
latest-ldap
latest-ldap-1.6.1
all feature tags 
```

I'm using `latest-ldap` productive and wanted to test the new beta via the `feature` tag. It would be nice if those get updated.

**Comment by jcollie** [comment]
With the new 2.0.0 images separate -ldap images are no longer being provided/updated as it's not necessary. In any case, discussion of the docker images would be better handled in the netbox-docker repository.
